extension/src/experiments/model/collect.ts

```diff
@@ -325,14 +310,14 @@ const hasCheckpoints = (data: ExpShowOutput) => {
 }
 
 export const collectExperiments = (
-  output: ExpShowOutput,
-  dvcLiveOnly: boolean,
-  commitsOutput: string | undefined
+  expShow: ExpShowOutput,
+  gitLog: string,
```

Comment:
[F] To me it makes more sense to put the `gitLog` output before `dvcLiveOnly` because it is more important for the data.